docs: upgraded `kurtosis feedback` command docs, added the `bug, fr and docs` flags

leoporoli opened the pull request:
## Description:
Upgraded `kurtosis feedback` command docs, added the `bug, fr and docs` flags

## Is this change user facing?
YES

## References (if applicable):
tickets: #226 and #219 

@leeederek you can review only this file: docs/docs/reference/cli/feedback.md because [there is another PR for the code](https://github.com/kurtosis-tech/kurtosis/pull/287)

leoporoli committed:
Adding bug, fr and docs flags

leoporoli committed:
Added mutually exclusive flags validation

leoporoli committed:
github by default, description edited

leoporoli committed:
Merge branch 'main' into lporoli/ticket-219

leoporoli committed:
Upgraded kurtosis feedback command docs

leoporoli committed:
Merge branch 'main' into lporoli/ticket-219-docs

leeederek commented on docs/docs/reference/cli/feedback.md:
```diff
@@ -17,18 +17,21 @@ which will return:
 where `YOUR_FEEDBACK` is the feedback you would like to send to us. 
 
-Running the above command (with no flags) will open a draft email using your default email client with the `TO:` line pre-filled with our feedback email list (feedback@kurtosistech.com) and the body of the message pre-filled with `YOUR_FEEDBACK` (that you entered in the command line). For example, `YOUR_FEEDBACK` could be: "_I enjoy the enclave names_".
+Running the above command (with no flags) will open the [Kurtosis Github choose new issue page](https://github.com/kurtosis-tech/kurtosis/issues/new/choose) where you can choose the issue type, and after that, when you press the button for selecting the type, a new page will be open with the description field pre-filled with `YOUR_FEEDBACK`
```
```suggestion
Running just `kurtosis feedback "my feedback"` (with no flags) will open the new issue creation page on [our Github](https://github.com/kurtosis-tech/kurtosis/issues/new/choose) where you can can select the issue type and have the description field pre-filled with `my feedback`. 
```

leeederek commented on docs/docs/reference/cli/feedback.md:
```diff
@@ -17,18 +17,21 @@ which will return:
 - The `--github` flag can be used to open the Issue creation page in our Github where you can select the Issue template you wish to use for your feedback. The `"$YOUR_FEEDBACK"` arg will be pre-populated in the description of whichever Issue template you select.
 - The `--email` flag opens a draft email to feedback@kurtosistech.com, via your default mail client, that has the body of the email pre-filled with whatever you entered in the `"$YOUR_FEEDBACK"` arg. This is the default behavior when no flag is set.
 - The `--calendly` flag can be used to open our [Calendly link](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding) to schedule time with our team to help you get started, address feedback, and answer any questions you may have.
+- The `--bug` flag indicate the `bug` feedback type, if it set along with the `github` flag the Github's new bug issue page will be open, and if it set along with the `email` flag, the email's client will be opened with a prefix text in the email's subject indicating the feedback's type
```
```suggestion
Additionally, the flags below can be used with `--github` and `--email` to specify the *type* of feedback you wish to provide:
- The `--bug` flag can be used when you wish to submit a bug report to us. When this `--bug` flag is set alongside the `--github` flag, the CLI will take you directly to the [bug report issue creation page](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=bug&template=bug-report.yml&title=%5Bbug%5D%3A+) in our Github. When this `--bug` flag is set alongside the `--email` flag, the CLI will open an email draft with the subject pre-filled with: `[BUG]`, which will help our team triage and prioritize your report.
```

leeederek replied:
Wanted to separate this section out because I wanted to make it clear that `--bug`, `--fr`, and `--docs` only has an affect on `--github` and `--email`, but not `--calendly`.

leeederek commented on docs/docs/reference/cli/feedback.md:
```diff
@@ -17,18 +17,21 @@ which will return:
 - The `--email` flag opens a draft email to feedback@kurtosistech.com, via your default mail client, that has the body of the email pre-filled with whatever you entered in the `"$YOUR_FEEDBACK"` arg. This is the default behavior when no flag is set.
 - The `--calendly` flag can be used to open our [Calendly link](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding) to schedule time with our team to help you get started, address feedback, and answer any questions you may have.
+- The `--bug` flag indicate the `bug` feedback type, if it set along with the `github` flag the Github's new bug issue page will be open, and if it set along with the `email` flag, the email's client will be opened with a prefix text in the email's subject indicating the feedback's type
+- The `--fr` flag indicate the `feature request` feedback type, if it set along with the `github` flag the Github's new feature request issue page will be open, and if it set along with the `email` flag, the email's client will be open with a prefix text in the email's subject indicating the feedback's type
```
```suggestion
- The `--fr` flag can be used when you wish to submit a feature request to us. When this `--fr` flag is set alongside the `--github` flag, the CLI will take you directly to the [feature request issue creation page](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=feature+request&template=feature-request.yml&title=%5BFR%5D%3A+) in our Github. When this `--fr` flag is set alongside the `--email` flag, the CLI will open an email draft with the subject pre-filled with: `[FEATURE_REQUEST]`, which will help our team triage and prioritize your request.
```

leeederek commented on docs/docs/reference/cli/feedback.md:
```diff
@@ -17,18 +17,21 @@ which will return:
 - The `--calendly` flag can be used to open our [Calendly link](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding) to schedule time with our team to help you get started, address feedback, and answer any questions you may have.
+- The `--bug` flag indicate the `bug` feedback type, if it set along with the `github` flag the Github's new bug issue page will be open, and if it set along with the `email` flag, the email's client will be opened with a prefix text in the email's subject indicating the feedback's type
+- The `--fr` flag indicate the `feature request` feedback type, if it set along with the `github` flag the Github's new feature request issue page will be open, and if it set along with the `email` flag, the email's client will be open with a prefix text in the email's subject indicating the feedback's type
+- The `--docs` flag indicate the `docs` feedback type, if it set along with the `github` flag the Github's new feature request issue page will be open, and if it set along with the `email` flag, the email's client will be open with a prefix text in the email's subject indicating the feedback's type
```
```suggestion
- The `--docs` flag can be used when you wish to flag an issue with our documentation. When this `--docs` flag is set alongside the `--github` flag, the CLI will take you directly to the [docs issue creation page](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=leeederek&labels=docs&template=docs-issue.yml&title=%5BDocs%5D%3A+) in our Github. When this `--docs` flag is set alongside the `--email` flag, the CLI will open an email draft with the subject pre-filled with: `[DOCS]`, which will help our team triage and prioritize the issue.
```

leeederek reviewed (approved):
Looks great! Thank you Leandro! Just had some minor reformatting proposals that you can commit before merging!

leoporoli committed:
Merge branch 'main' into lporoli/ticket-219

leoporoli committed:
Modified -fr flag

leoporoli committed:
Merge branch 'lporoli/ticket-219' into lporoli/ticket-219-docs

leeederek committed:
fix: name all args for add_services instruction in quickstart (#316)

## Description:
Since Starlark requires named args to come after positional args, or
have all args be named, the change made
[here](https://github.com/kurtosis-tech/kurtosis/pull/299/files#diff-db23839e9fa71374d53983730e318028367dff34e8c817cd1e8acfec09624fe8L200)
meant that the starlark package would fail to run.

This PR names _all_ the args in the `add_service` and `add_services`
instruction in our quickstart code snippets to ensure it runs smoothly.
This change also ensures that the code snippet follows the same syntax
structure as what our reference docs say
[here](https://docs.kurtosis.com/reference/starlark-instructions#add_service)

## Is this change user facing?
YES
<!-- If yes, please add the "user facing" label to the PR -->
<!-- If yes, don't forget to include docs changes where relevant -->

## References (if applicable):
<!-- Add relevant Github Issues, Discord threads, or other helpful
information. -->
#299

---------

Co-authored-by: Derek Lee <derek@Dereks-MacBook-Pro.local>

galenmarchetti committed:
docs: highlight path through quickstart (#312)

## Description:
<!-- Describe this change, how it works, and the motivation behind it.
-->
Docs polish 

## Is this change user facing?
YES
<!-- If yes, please add the "user facing" label to the PR -->
<!-- If yes, don't forget to include docs changes where relevant -->

## References (if applicable):
<!-- Add relevant Github Issues, Discord threads, or other helpful
information. -->

leoporoli committed:
Added replace in core/server go.mod file

leoporoli committed:
Update docs/docs/reference/cli/feedback.md

Co-authored-by: Derek Lee <103802618+leeederek@users.noreply.github.com>

leoporoli committed:
Update docs/docs/reference/cli/feedback.md

Co-authored-by: Derek Lee <103802618+leeederek@users.noreply.github.com>

leoporoli committed:
Update docs/docs/reference/cli/feedback.md

Co-authored-by: Derek Lee <103802618+leeederek@users.noreply.github.com>

leoporoli committed:
Update docs/docs/reference/cli/feedback.md

Co-authored-by: Derek Lee <103802618+leeederek@users.noreply.github.com>

leoporoli committed:
Renamed --fr flag

leoporoli committed:
Merge branch 'main' into lporoli/ticket-219-docs

leoporoli committed:
Merge branch 'main' into lporoli/ticket-219-docs
